src/client.js

```diff
@@ -1764,6 +1768,44 @@ class Client {
         return result;
     }
 
+    /**
+     * This is the exposed/public method to request context data for a specific report.
+     * @param {*} callContext: {reportName: string, context: string, selection: string, selectionCount: number, formData: any}
+     * @param {string} representation the expected representation ('xml', 'BadgerFish', or 'SimpleJson'). If not set, will use the current representation
+     *
+     * @returns {Campaign.ReportContext} an object containing the context data for a specific report
+     */
+    async getReport(callContext, representation) {
```

Comment:
Can we call this getReportData as it's returning the data and not the report itself